### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by DivyanshSareen**
added product listing page. updated category card

homepage: https://deploy-preview-2--pandapanda.netlify.app/
product listing page: https://deploy-preview-2--pandapanda.netlify.app/product-list.html

### CODE REVIEW

**Review by zoyatanweer — APPROVED** [review]
looks good to merge

**Review by sanjitsarkar — APPROVED** [review]
looks good to be merged

**Comment by zoyatanweer** [comment]
Code is good, few suggestions on UI:
1. On the homepage, there is no direct link to "products" page. You can create <a> or <button> for that.
2. On the products page, in the top left navbar, your "ecom store" is in blue color. Kindly change it to the color it is on the homepage.
3. Write something in the placeholder of the search bar. Ex: Look for products etc.
4. Make your cart and wishlist badge bigger in size.

File: product-list.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+.filter{
+    width: 18em;
+    padding: 1em;
+    box-shadow: var(--list-shadow);
+    position: sticky;
+}
+
+.filter-and-clear{
+    display: flex;
+    flex-direction: row;
```
Comment (zoef-yt) [inline product-list.css]: flex is by default row if you are not overriding this then it can be removed.

File: product-list.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.price > .input-data{
+    display: block;
+    width: 100%;
+}
+
+.price-values{
+    display: flex;
+    flex-direction: row;
```
Comment (zoef-yt) [inline product-list.css]: flex is by default row if you are not overriding this then it can be removed.

**Review by purvasharmadev — APPROVED** [review]
looks good to merge.

**Review by zoef-yt — APPROVED** [review]
Clean code, Easy to understand. Nice that you added pre-view links just a suggestion adding a screenshot would make is great. With the minor suggested changes, it's good to merge. Also +1 to the UI suggestion mentioned by Zoya.

**Comment by purvasharmadev** [comment]
> Code is good, few suggestions on UI:
> 
> 1. On the homepage, there is no direct link to "products" page. You can create  or  for that.
> 2. On the products page, in the top left navbar, your "ecom store" is in blue color. Kindly change it to the color it is on the homepage.
> 3. Write something in the placeholder of the search bar. Ex: Look for products etc.
> 4. Make your cart and wishlist badge bigger in size.

+1
> 5. On the homepage, "badges" on the product card are hard to notice, try to make it a little bit bigger.

**Comment by DivyanshSareen** [comment]
I have made some fixes based on the reviews by Zoya and Zoef, will fix the blue links in a later PR.

**Pull request merged**